recipe/meta.yaml

```diff
@@ -23,47 +23,47 @@ requirements:
     - blas 1.1 {{ variant }}  # [not win]
     - libnetcdf 4.4.*
     - proj4 4.9.3
-    - libgdal 2.2.*
+    - libgdal 2.1.*
     - g2clib 1.6.*
     - hdfeos2
     - hdfeos5
     - jasper
-    - libpng >=1.6.28,<1.7
+    - libpng >=1.6.22,<1.6.31
     - jpeg 9*
-    - zlib 1.2.8
+    - zlib 1.2.11
     - hdf4
     - hdf5 1.8.18|1.8.18.*
```

Comment:
We moved to `hdf5 1.10.1` but I was unable to build `hdfeos5` with it. So I guess we should keep `1.8.18` here for a while.